src/Components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import SearchIcon from "@mui/icons-material/Search";
+import ClearIcon from "@mui/icons-material/Clear";
+import { SearchBarProps } from "./types";
+import Input from "../Input/Input";
+import { endStyle, inputBaseStyle, paperStyle, startStyle } from "./styles";
+
+const SearchBar: React.FC<SearchBarProps> = ({
+  placeHolder = "Search...",
+  disabled = false,
+  onSearch,
+  onClear,
+}) => {
+  const [inputValue, setInputValue] = useState("");
+  const [isInputBaseFouced, setIsPressed] = useState<boolean>(false);
```

Comment:
change to `const [isInputFocused, setIsInputFocused] = useState<boolean>(false)`
to maintain react useState convention